### DISCUSSION TITLE
File Transfer From UTM Windows 11 VM to macOS folder with Sharing.

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#3578] [UTM Help (macOS)]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration**
* UTM Version: 
* OS Version: 
* Intel or Apple Silicon? 

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by ghost** [comment]
I have MacBook Air M1 with 512 GB SSD and 8GB RAM. I installed Windows 11 as UTM vm from .qcow. I used shared folder setting to connect host os files. But copy from guest to host is not taking place. Net is working with spices tools. What to do to enable file transfer from guest to host.

**Reply by tfig-dev** [reply]
I solved it by create a shared folder with read/write permission for everyone within Windows (guest), I enabled the detection of public folders and I access it by SMB on the Mac (host). Feel free to ask anything

**Reply by Rakscreator** [reply]
Can you drop a tutorial for the same?

**Reply by tfig-dev** [reply]
here's a free Youtube video: https://www.youtube.com/watch?v=RYYe_Y-99fU

**Reply by Remus1992** [reply]
For anyone else who comes here: here's another helpful video on the matter: https://www.youtube.com/watch?v=UogQu1S0p2Y&ab_channel=TouchTechnologyReview
